### DISCUSSION TITLE
Is the latest code on Develop updated to handle the new Integrity check?

### DISCUSSION THREAD

**Discussion by BarkerB** [RakambdaOrg/ChannelPointsMiner#288] [Q&A] [answered]
Just wondering if Develop is able to handle Twitch's new integrity check? I set up the app for the first time today, and it's running and scraping info from the chat and everything but is not successfully placing predictions OR claiming the +50 points every 15 minutes. The predictions could be an issue with configuration on my end as I'm still trying to parse through the documentation, but I believe it should be claiming the +50 regularly regardless?

Edit: Once I set a logger to filter out the trace messages, I now see the error response confirming the integrity check failing, my bad if this was already a known problem
![image](https://user-images.githubusercontent.com/10618329/190321032-b867aedb-6b38-4123-baff-71a0462d4874.png)

**Comment by Rakambda** [comment, marked as answer]
I tried handling that yesterday, but apparently it isn't enough. Will get back at it after work.

#285

**Reply by Rakambda** [reply]
Seems to be fine on Dev for now, will let it run for a bit to be sure and then release it officially
